models/database.go

```diff
@@ -46,23 +46,23 @@ var databaseSchema = [][]string{
 			node_id VARCHAR(255) NOT NULL,
 			node_type VARCHAR(255) NOT NULL,
 			node_name VARCHAR(255) NOT NULL,
-			machine_id VARCHAR(255),
-			custom_labels TEXT,
-			address VARCHAR(255), -- also RDS instance
+			machine_id VARCHAR(255), -- NULL means "unknown"; non-NULL value must be unique
+			custom_labels TEXT, -- NULL for nil []byte
+			address VARCHAR(255) NOT NULL, -- also Remote instance
 			created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
 			-- updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
 
 			-- Generic
-			distro VARCHAR(255),
-			distro_version VARCHAR(255),
+			distro VARCHAR(255) NOT NULL,
+			distro_version VARCHAR(255) NOT NULL,
 
 			-- Container
-			docker_container_id VARCHAR(255),
-			docker_container_name VARCHAR(255),
+			docker_container_id VARCHAR(255), -- NULL means "unknown"; non-NULL value must be unique
+			docker_container_name VARCHAR(255) NOT NULL,
 
 			-- RemoteAmazonRDS
-			-- instance is address
-			region VARCHAR(255),
+			-- RDS instance is stored in address
+			region VARCHAR(255), -- NULL means "not Remote"; non-NULL value must be unique in combination with instance/address
```

Comment:
@AlekSi Btw I think it will be better to add comments inside the schema so you can see this info inside Table Info
```suggestion
			region VARCHAR(255) COMMENT 'NULL means "not Remote"; non-NULL value must be unique in combination with instance/address',
```